T23208: really important message
CallStack (from HasCallStack):
  error, called at T23208_Lib.hs:4:7 in main:T23208_Lib
HasCallStack backtrace:
CallStack (from HasCallStack):
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:89:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:125:3 in ghc-internal:GHC.Internal.Exception


